src/main/java/uk/gov/register/presentation/dao/EntryDAO.java

```diff
@@ -17,6 +18,10 @@
     @SingleValueResult(Entry.class)
     Optional<Entry> findByEntryNumber(@Bind("entry_number") int entryNumber);
 
+    @SqlQuery("select * from entry limit 100")
+    @RegisterMapper(NewEntryMapper.class)
+    List<Entry> getAll();
```

Comment:
why do you think so? is it because of there is a limit of 100 entries in query? 
Any other name preference? I can rename it to like `getEntries`.